safe-global/safe-wallet-web
# Move from ethereum-blockies-base64 to blo

## What it solves

Makes the generation of identicons faster and more precise, while reducing the bundle size.

Disclaimer: I am the author of blo.

## How this PR fixes it

By moving to [blo](https://github.com/bpierre/blo).

## How to test it

Run the app and check anywhere the `Identicon` component gets rendered. See also this other PR: https://github.com/safe-global/safe-react-components/pull/229

## Screenshots

Before:
<img src=https://github.com/safe-global/safe-wallet-web/assets/36158/a4c5decc-48d7-4702-bdf3-27fb4358d7b0 width=400>


After:
<img src=https://github.com/safe-global/safe-wallet-web/assets/36158/0b95d4e3-ee27-4975-a15d-10a685f1d3d5 width=400>

Before (zoomed in):
<img src=https://github.com/safe-global/safe-wallet-web/assets/36158/9f0585d6-0395-40c7-b0ad-d34165185a13 width=200>

After (zoomed in):
<img src=https://github.com/safe-global/safe-wallet-web/assets/36158/bfa9d3f7-68a0-4c49-8775-2c9313e09bcc width=200>


## Checklist
